applications/luci-app-frpc/htdocs/luci-static/resources/view/frpc.js

```diff
@@ -115,12 +116,68 @@ function defOpts(s, opts, params) {
 	}
 }
 
+var conf = 'frpc';
```

Comment:
I would not move this into its own var.
From my point of view, that makes no sense.
This is not changed at runtime.

Reply:
yeap, just make functions more versatile. I'll remove it.